pyvo/io/uws/__init__.py

```diff
@@ -1,4 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-__all__ = ['parse_job', 'JobFile']
+__all__ = ['parse_job', 'parse_for_object', 'JobFile', 'JobList']
 
-from .endpoint import parse_job, JobFile
+from .endpoint import *
+from .tree import *
```

Comment:
It seems like your doc build is broken, I'm no sphinx expert at all, but it looks like it's complaining about uwstree, and it might be just walking everything that's imported.  But the error looks strange and I don't know what it really means.

Reply:
This is a bugger. I can't reproduce it locally... I'll keep an eye on this - thanks for the tip.